### PR TITLE
Cleanup global DScript names

### PR DESCRIPTION
**Pull request by dannyvv**

### CODE REVIEW

File: Public/Src/FrontEnd/Script/Evaluator/CommonConstants.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    public sealed class CommonConstants
+    {
+        /// <nodoc />
+        public SymbolAtom Qualifier { get; }
+
+        /// <nodoc />
+        public SymbolAtom Length { get; }
+
+        /// <nodoc />
+        private readonly StringTable m_stringTable;
```
Comment (SergeyTeplyakov) [inline Public/Src/FrontEnd/Script/Evaluator/CommonConstants.cs]: Very nit: move it on the top:)

File: Public/Src/FrontEnd/Script/Evaluator/ContextTree.cs [hunk 2]
```diff
@@ -44,6 +44,11 @@ public sealed class ContextTree : IDisposable
         /// </summary>
         public GlobalConstants Constants { get; private set; }
 
+        /// <summary>
+        /// CommonConstants
```
Comment (SergeyTeplyakov) [inline Public/Src/FrontEnd/Script/Evaluator/ContextTree.cs]: nodoc?

Comment (SergeyTeplyakov) [reply Public/Src/FrontEnd/Script/Evaluator/ContextTree.cs]: Or at least "Common constants" :)))

---
In reply to: [274539409](https://github.com/Microsoft/BuildXL/pull/127#discussion_r274539409) [](ancestors = 274539409)

Comment (dannyvv) [reply Public/Src/FrontEnd/Script/Evaluator/ContextTree.cs]: will do in next PR

File: Public/Src/FrontEnd/Script/Evaluator/CommonConstants.cs (hunk 1 above)
Comment (dannyvv) [reply Public/Src/FrontEnd/Script/Evaluator/CommonConstants.cs]: Will do no next PR

---
In reply to: [274539264](https://github.com/Microsoft/BuildXL/pull/127#discussion_r274539264) [](ancestors = 274539264)

**Pull request merged**